homeassistant/components/zha/sensor.py

```diff
@@ -838,6 +838,26 @@ class SmartEnergySummationReceived(PolledSmartEnergySummation):
     _unique_id_suffix = "summation_received"
     _attr_translation_key: str = "summation_received"
 
+    @classmethod
+    def create_entity(
+        cls,
+        unique_id: str,
+        zha_device: ZHADevice,
+        cluster_handlers: list[ClusterHandler],
+        **kwargs: Any,
+    ) -> Self | None:
+        """Entity Factory.
+
+        This attribute only started to be initialized in HA 2024.2.0,
+        so the entity would still be created on the first HA start after the upgrade for existing devices,
```

Comment:
Please limit docstring lines to 72 characters per PEP8.

Reply:
https://github.com/home-assistant/core/pull/109774